feature/system/gnmi/cliorigin/ate_tests/mixed_oc_cli_origin_support_test/README.md

```diff
@@ -39,7 +39,7 @@ required in the emergency case that is covered by this requirement.
     *   `origin: ""` (openconfig, default origin) setting the DUT port-1 string
         value at `/interfaces/interface/config/description` to `"from oc"`.
 
-*   Validate that DUT port-1 description is `"from oc"`.
+*   Validate that DUT port-1 description is `"from cli"` as a CLI value should take precedence over an OpenConfig value if both are defined as per [Mixing Schemas in gNMI](https://github.com/openconfig/reference/blob/master/rpc/gnmi/mixed-schema.md).
```

Comment:
Where in the document does it say this precedence? I scanned the doc but couldn't find it.

Reply:
Removed interface description tests and so also removed this.